roles/edpm_nova_compute/molecule/default/converge.yml

```diff
@@ -17,5 +17,29 @@
 
 - name: Converge
   hosts: all
+  vars:
+    edpm_container_standalone_service: test
+    edpm_container_standalone_container_defs:
+      test:
+        image: quay.io/centos/centos:stream9
+        net: host
+        privileged: true
+        restart: always
+        environment:
+            KOLLA_CONFIG_STRATEGY: COPY_ALWAYS
+    edpm_container_standalone_kolla_config_files:
+      test:
+        command: 'sleep 3600'
+  pre_tasks:
+    - ansible.builtin.set_fact:
+        internal_api:
+          nova_api_network: mock_network
+          nova_libvirt_network: mock_network
+        internal_api_ip: 10.0.0.1
+        fqdn_internal_api: mock_api
   roles:
+    - role: "osp.edpm.edpm_container_standalone"
```

Comment:
ok so this role is setting some default in teh env.

i would generaly consider that to be a bug in the interaction between these two roles but i guess that is ok for now.